packages/siopv2-oid4vp-rp-rest-api/src/SIOPv2RPRestAPI.ts

```diff
@@ -241,14 +239,16 @@ export class SIOPv2RPRestAPI {
             ],
           })
           .then((verifiedResponse: VerifiedAuthorizationResponse) => {
-            // console.log('verifiedResponse: ', JSON.stringify(verifiedResponse, null, 2))
-
             const wrappedPresentation = verifiedResponse?.oid4vpSubmission?.presentations[0]
             if (wrappedPresentation) {
               const credentialSubject = wrappedPresentation.presentation.verifiableCredential[0].credential.credentialSubject
-              console.log('AND WE ARE DONE!')
-              console.log(JSON.stringify(credentialSubject, null, 2))
-              console.log(JSON.stringify(wrappedPresentation.presentation, null, 2))
+              debug(
```

Comment:
Please revert this one. This module is still being developed and tested. I will do a pass once it is ready